packages/nds-stacked-nav/scss/stacked-nav.scss

```diff
@@ -41,7 +54,7 @@
     position: relative;
     text-decoration: none;
 
-    &[aria-current='page'] {
+    &[aria-current='true'] {
```

Comment:
Can we add both aria-current='true' and aria-current='page' in here for non-react cases where might use either attribute. Page being the exact page you're on and true being just active in general (e.g. ancestor)